package/rmt-server-sync.timer

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=RMT Sync timer
+After=rmt.service
+Requires=rmt.service
+
+[Timer]
+OnCalendar=*-*-* 01:00:00
```

Comment:
All RMTs in the same timezone will bomb our servers at the same time! Self-inflicted DDoS FTW!

Maybe we can randomize this.

Reply:
🙏 please randomize :-), or at least not at the same time as all SMTs (they also DDoS us at 1:00). 

Using something like `H H * * *` might be good.

Reply:
I put in a random delay of 3 hours